components/shopify/actions/create-order/create-order.mjs

```diff
@@ -0,0 +1,142 @@
+import shopify from "../../shopify.app.mjs";
+import { toSingleLineString } from "../commons.mjs";
+
+export default {
+  key: "shopify-create-order",
+  name: "Create Order",
+  description: toSingleLineString(`Creates a new order.
+    For full order object details [see the docs](https://shopify.dev/api/admin-rest/2022-01/resources/order#[post]/admin/api/2022-01/orders.json)
+    or [see examples](https://shopify.dev/api/admin-rest/2022-01/resources/order#[post]/admin/api/#{api_version}/orders.json_examples)
+  `),
+  version: "0.0.1",
+  type: "action",
+  props: {
+    shopify,
+    lineItems: {
+      type: "string[]",
+      label: "Line Items",
+      description: toSingleLineString(`
+        A list of line item objects, each containing information about an item in the order.
+        More details when searching **line_items** in [Shopify Order Object](https://shopify.dev/api/admin-rest/2022-01/resources/order#resource_object)
```

Comment:
Add an example for this prop's value in the description? For example:
```js
      description: toSingleLineString(`
        A list of line item objects, each containing information about an item in the order (e.g.,
        \`{"variant_id":447654529,"quantity":1}\`). More details when searching **line_items** in
        [Shopify Order
        Object](https://shopify.dev/api/admin-rest/2022-01/resources/order#resource_object)
      `),
```

Reply:
Issue addressed